lib/compat/wordpress-5.9/class-wp-theme-json-schema-gutenberg.php

```diff
@@ -77,6 +77,10 @@ public static function migrate( $theme_json ) {
 			$theme_json = self::migrate_v1_to_v2( $theme_json );
 		}
 
+		if ( 2 === $theme_json['version'] ) {
+			$theme_json['version'] = 3;
+		}
```

Comment:
I appreciate this change is trivial but for consistency with the other theme.json classes and 6.0 related changes should we override this class in the same manner? I'm also ok with leaving it here.

Reply:
oh, you're right, should this land, we need to fix it.